### ISSUE TITLE
Impossible de supprimer une vieille sauvegarde

### ISSUE THREAD

**Issue by Fouindor** [JulienNigon/SpellcasterUniversity-Bug-Report#1009]
Hello,
Je n'arrive pas a supprimer la plus vieille sauvegarde, "Wololo" a partir du jeu. Les autres ne sont pas affectées, je peux les supprimer.
![20200415201333_1](https://user-images.githubusercontent.com/15092739/79373091-85fbad00-7f56-11ea-8363-c716b1621a8b.jpg)

Ça n'a pas l'air d'être dû au nombre de sauvegardes. Quand je supprime manuellement d'autres sauvegardes, je ne peux toujours pas supprimer "Wololo".

Ça n'a pas l'air dû au fait que ce soit la dernière sauvegarde: quand je supprime "Wololo", je peux supprimer toutes les autres.

Ci-joint Wololo:
[Wololo.zip](https://github.com/JulienNigon/SpellcasterUniversity-Bug-Report/files/4482990/Wololo.zip)

Bon courage :)

**Comment by Neelan22** [comment]
Tu ne peux pas la supprimer en allant la chercher directement dans ton dossier saves?
Le dev est en train de mettre en place une sécurité avec les sauvegardes. Ca devrait être effectif à la prochaine MaJ

**Comment by Fouindor** [comment]
Oui, je peux la supprimer en allant dans le dossier, c'est ce que je voulais dire quand je disais "supprimer manuellement". C'est pas fou comme soucis mais je préférais le remonter au cas ou :)

**Comment by JulienNigon** [comment]
Tu as bien fait :)
